mleap-runtime/src/main/scala/ml/combust/mleap/bundle/ops/clustering/LDAModelOp.scala

```diff
@@ -0,0 +1,65 @@
+package ml.combust.mleap.bundle.ops.clustering
+
+import breeze.linalg.{Matrix, DenseMatrix => BDM, DenseVector => BDV}
+import ml.combust.bundle.BundleContext
+import ml.combust.bundle.dsl._
+import ml.combust.bundle.op.{OpModel, OpNode}
+import ml.combust.mleap.core.clustering.LocalLDAModel
+import ml.combust.mleap.runtime.MleapContext
+import ml.combust.mleap.runtime.transformer.clustering.LDAModel
+import ml.combust.mleap.tensor.DenseTensor
+
+/**
+  * Created by mageswarand on 3/3/17.
+  */
+class LDAModelOp extends OpNode[MleapContext, LDAModel, LocalLDAModel] {
+
+  override val Model: OpModel[MleapContext, LocalLDAModel] = new OpModel[MleapContext, LocalLDAModel] {
+    override val klazz: Class[LocalLDAModel] = classOf[LocalLDAModel]
+
+    override def opName: String = Bundle.BuiltinOps.clustering.lda
+
+    override def store(model: Model, obj: LocalLDAModel)(implicit context: BundleContext[MleapContext]): Model = {
+      val topicMatrixArray = obj.topicsMatrix.toDenseMatrix.toArray //TODO should we add SparseMatrix?
```

Comment:
Can always store as Sparse matrix as an array of indices and an array of values if this is likely to cause issues.